cmake/config.cuh.in

```diff
@@ -0,0 +1,22 @@
+/*
+*  Copyright 2021 NVIDIA Corporation
+*
+*  Licensed under the Apache License, Version 2.0 with the LLVM exception
+*  (the "License"); you may not use this file except in compliance with
+*  the License.
+*
+*  You may obtain a copy of the License at
+*
+*      http://llvm.org/foundation/relicensing/LICENSE.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+
+#pragma once
+
+// Defined if NVBench has been built with NVML support.
+#cmakedefine NVBENCH_HAS_NVML
```

Comment:
I don't see where NVBENCH_HAS_NVML is set. I do see NVBENCH_ENABLE_NVML

Reply:
It's in nvbench/cmake/NVBenchWriteConfigHeader.cmake:

```
function(nvbench_write_config_header filepath)
  if (NVBench_ENABLE_NVML)
    set(NVBENCH_HAS_NVML 1)
  endif()

  configure_file("${NVBench_SOURCE_DIR}/cmake/config.cuh.in" "${filepath}")
endfunction()
```